orb_models/forcefield/pretrained.py

```diff
@@ -131,28 +130,296 @@ def orb_v2_architecture(
+    activation: str = "silu",
+    device: Optional[torch.device] = None,
+) -> DirectForcefieldRegressor:
+    """The orb-v3 architecture, defaulting to a direct model."""
+    model = DirectForcefieldRegressor(
+        heads={
+            "energy": EnergyHead(
+                latent_dim=latent_dim,
+                num_mlp_layers=head_mlp_depth,
+                mlp_hidden_dim=head_mlp_hidden_dim,
+                predict_atom_avg=True,
+                activation=activation,
+            ),
+            "forces": ForceHead(
+                latent_dim=latent_dim,
+                num_mlp_layers=head_mlp_depth,
+                mlp_hidden_dim=head_mlp_hidden_dim,
+                remove_mean=True,
+                remove_torque_for_nonpbc_systems=True,
+                activation=activation,
+            ),
+            "stress": StressHead(
+                latent_dim=latent_dim,
+                num_mlp_layers=head_mlp_depth,
+                mlp_hidden_dim=head_mlp_hidden_dim,
+                node_aggregation="mean",
+                activation=activation,
+            ),
+            "confidence": ConfidenceHead(
+                latent_dim=latent_dim,
+                num_mlp_layers=head_mlp_depth,
+                mlp_hidden_dim=head_mlp_hidden_dim,
+                activation=activation,
+            ),
+        },
+        model=MoleculeGNS(
+            latent_dim=latent_dim,
+            num_message_passing_steps=num_message_passing_steps,
+            num_mlp_layers=base_mlp_depth,
+            mlp_hidden_dim=base_mlp_hidden_dim,
+            rbf_transform=BesselBasis(
+                r_max=6.0,
+                num_bases=8,
+            ),
+            angular_transform=SphericalHarmonics(
+                lmax=3,
+                normalize=True,
+                normalization="component",
+            ),
+            outer_product_with_cutoff=True,
+            use_embedding=True,
+            interaction_params={
+                "distance_cutoff": True,
+                "attention_gate": "sigmoid",
+            },
+            node_feature_names=["feat"],
+            edge_feature_names=["feat"],
+            activation=activation,
+            mlp_norm="rms_norm",
+        ),
+        pair_repulsion=True,
+    )
+    if device is not None and device != torch.device("cpu"):
+        model.cuda(device)
+    else:
+        model = model.cpu()
+
+    return model
+
+def orb_v3_conservative_20_omat(
+    weights_path: str = "",  # noqa: E501
+    device: Union[torch.device, str, None] = None,
+    precision: str = "float32-high",
+    compile: Optional[bool] = None,
+) -> Tuple[ConservativeForcefieldRegressor, SystemConfig]:
+    """Load ORB v3 Conservative 20 max neighbors OMAT."""
+    model = orb_v3_conservative_architecture(device=device)
+    model = load_model_for_inference(
+        model, weights_path, device, precision=precision, compile=compile
+    )
+
+    return model, SystemConfig(radius=6.0, max_num_neighbors=20)
```

Comment:
Given that we expect people to use these constructor functions, does it make sense to add the system config to the model in some way such that we don't have to pass around 2 objects? I know we do this internally, but it would make it easier for external users - e.g a `model.system_config` or `model.max_num_neighbors` which we could then use as a default in the calculator.

Reply:
This makes using models easier for people who don't care about the details, so I went ahead and implemented it